core/src/commonMain/kotlin/com/revenuecat/purchases/kmp/Package.kt

```diff
@@ -67,17 +67,3 @@ public expect enum class PackageType {
      */
     WEEKLY,
 }
-
-public val PackageType.identifier: String?
```

Comment:
I think this isn't needed by consumers. I know we expose it from Android (not iOS), but feels like something we should hide there instead.